src/ui/components/Library/ViewerRouter.tsx

```diff
@@ -65,14 +65,14 @@ function NonPendingTeamLibrary({ currentWorkspaceId }: ViewerRouterProps) {
 
   const workspace = workspaces.find(ws => ws.id === currentWorkspaceId)!;
 
-  return (
-    <Viewer
-      recordings={recordings}
-      workspaceName={
-        workspace.logo ? <Base64Image src={workspace.logo} className="max-h-12" /> : workspace.name
-      }
-    />
+  const workspaceName = (
+    <div className="flex flex-row space-x-2">
```

Comment:
If we're adding it, let's add `items-center` too so everything will be aligned reasonably